### PR TITLE
feat: Coralogix Destination

### PR DESCRIPTION
**Pull request by cpheps**
### Proposed Change
Added a destination for the Coralogix eporter that is newly available in collector version `v1.13.0`.

I validated sending metrics and logs to a Coralogix instance.

##### Checklist
- [x] Changes are tested
- [ ] CI has passed

### CODE REVIEW

File: docs/www/integrations/destinations/coralogix.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+| enable_logs                 | `bool`    | true                             |                                                                                                                                                                             |
+| log_endpoint                | `string`  | `otel-logs.coralogix.com:443`    | Coralogix Logs ingress endpoint                                                                                                                                             |
+| enable_traces               | `bool`    | true                             |                                                                                                                                                                             |
+| trace_endpoint              | `string`  | `otel-traces.coralogix.com:443`  | Coralogix Traces ingress endpoint                                                                                                                                           |
```
Comment (jsirianni) [inline docs/www/integrations/destinations/coralogix.md:24]: It would be nice to link to the endpoint docs here and in the parameter(s) using docurl.

**Review by jsirianni — APPROVED** [review]
One small comment, but it is working great.

Screenshot from 2022-12-02 11-11-17

Screenshot from 2022-12-02 11-11-42

Screenshot from 2022-12-02 11-12-04

File: docs/www/integrations/destinations/coralogix.md (hunk 1 above)
Comment (cpheps) [reply docs/www/integrations/destinations/coralogix.md:24]: I didn't know there were endpoint docs. I'll take a look.

Comment (cpheps) [reply docs/www/integrations/destinations/coralogix.md]: @jsirianni Updated to use a region field like we talked about

**Review by jsirianni — APPROVED** [review]

**Review by Mrod1598 — APPROVED** [review]
image
 got it working

image

**Pull request merged**